glib/tests/value.rs

```diff
@@ -0,0 +1,13 @@
+use glib::*;
+
+/// Test that `ToValue` (and conversely, `FromValue`) uphold the promised invariants
+#[test]
+pub fn to_value_invariants() {
+    // Inverse
+    assert_eq!(0i32, 0i32.to_value().get().unwrap());
```

Comment:
```suggestion
    assert_eq!(0i32, 0i32.to_value().get::<i32>().unwrap());
```

Reply:
I don't understand why this is required. The test compiles for me without it. But I've also seen the CI fail on this, and that's confusing.

Reply:
Yeah also it happens only on that one Windows job for some reason... not sure, it shouldn't be required at all.